cmd/apps/traefik2.go

```diff
@@ -0,0 +1,164 @@
+package apps
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	execute "github.com/alexellis/go-execute/pkg/v1"
+	"github.com/morikuni/aec"
+	"github.com/spf13/cobra"
+)
+
+func installTraefik2(parts ...string) (execute.ExecResult, error) {
+
+	task := execute.ExecTask{
+		Command:     "helm",
+		Args:        parts,
+		StreamStdio: true,
+	}
+	res, err := task.Execute()
+	if err != nil {
+		return res, err
+	}
+	if res.ExitCode != 0 {
+		return res, fmt.Errorf("exit code %d, stderr: %s", res.ExitCode, res.Stderr)
+	}
+	return res, nil
+}
+
+func MakeInstallTraefik2() *cobra.Command {
+	var traefik2 = &cobra.Command{
+		Use:          "traefik2",
+		Short:        "Install traefik2",
+		Long:         "Install traefik2",
+		Example:      `  arkade app install traefik2`,
+		SilenceUsage: true,
+	}
+
+	traefik2.Flags().StringP("namespace", "n", "kube-system", "The namespace used for installation")
+	traefik2.Flags().Bool("update-repo", true, "Update the helm repo")
+	traefik2.Flags().Bool("dashboard", false, "Expose dashboard if you want access to dashboard from the browser")
```

Comment:
@braybaut Have you tried out `traefik.Flags().BoolVar()` that way we can use `--dashboard` alone to set the value and not `--dashboard=true`. But it's totally up to you what you like to do, just a suggestion.

Reply:
I liked this approach, I changed  to BoolVar()  and now the flag is --dashboard